### ISSUE TITLE
Warning: Use Cipheriv for counter mode of aes-256-ctr

### ISSUE THREAD

**Issue by alexilyaev** [tkalfigo/dotenvenc#1]
I'm getting this warning:

```
$ npx dotenvenc -e $DOTENVENC_KEY
(node:7361) Warning: Use Cipheriv for counter mode of aes-256-ctr
```

**Comment by normancarcamo** [comment]
Me too here...

**Comment by tkalfigo** [comment]
Thanks for the patience and apologies for the very long delay in addressing this.

This is resolved in [v2.0.0](https://github.com/tkalfigo/dotenvenc/releases/tag/v2.0.0)
